rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java

```diff
@@ -24,8 +24,8 @@
 
     /**
      * A function that returns this operation as an Observable.
-     * When the returned observable will be subscribed this operation will be scheduled
-     * to be run on the main thread in future. When appropriate the call to run() will be executed.
+     * When the returned observable is subscribed this operation will be scheduled
```

Comment:
missing semicolon after `subscribed`?

Reply:
That's correct, there is a dependency between sentences.